android/PhysicalWeb/app/src/main/res/values/strings.xml

```diff
@@ -21,16 +20,23 @@
     <string name="title_about">About</string>
     <string name="empty_nearby_beacons_list_text">Searching for beacons</string>
     <string name="about_app_title">Physical Web</string>
-    <string name="url_getting_started">        https://google.github.io/physical-web/mobile/android/getting-started.html</string>
+    <string name="url_getting_started">https://google.github.io/physical-web/mobile/android/getting-started.html</string>
     <string name="config_no_beacons_found">No beacons found</string>
     <string name="notification_view_more_beacons">VIEW MORE</string>
     <string name="config_url_read_error">Error Reading URL</string>
-
+    <string name="bt_on">Turn Bluetooth on to broadcast</string>
+    <string name="ble_os_error">Bluetooth LE advertising is not supported by your OS</string>
+    <string name="no_url_error">Couldn\'t find URL to broadcast</string>
+    <string name="ble_error">Bluetooth LE advertising is not supported on this device</string>
+    <string name="nonascii_error">Sorry we don\'t support non-ascii URLs</string>
+    <string name="url_broadcast">URL is broadcasting!</string>
+    <string name="broadcast_notif"> Physical Web is broadcasting a URL</string>
```

Comment:
The leading space will get ignored and won't show up in the notification, but could you remove it anyway?